gatsby-node.ts

```
@@ -92,9 +92,9 @@ export const createPages: GatsbyNode['createPages'] = async ({
         }
     `);
 
-    const allCaseStudies = caseStudyPages.data.allStrapiCaseStudy.nodes;
+    const allCaseStudies = caseStudyPages.data?.allStrapiCaseStudy.nodes;
 
-    allCaseStudies.forEach((node) => {
+    allCaseStudies?.forEach((node) => {
```

We should really throw an error if this is empty so the build fails and we can catch issue with data missing that was expect.

---

This applies to a lot of the data below

---

Makes sense